prebuilts/featbit-data-analytics-server.toml

```diff
@@ -14,9 +14,6 @@ port = 80
 type = "HTTP"
 
 [env]
-REDIS_HOST = { default = "${REDIS_HOST}" }
-REDIS_PASSWORD = { default = "${REDIS_PASSWORD}" }
-REDIS_PORT = { default = "${REDIS_PORT}" }
 MONGO_URI = { default = "${MONGO_CONNECTION_STRING}" }
 MONGO_INITDB_DATABASE = { default = "featbit" }
 MONGO_HOST = { default = "mongodb.zeabur.internal" }
```

Comment:
I afraid if this is also unnecessary.

Reply:
Yes, it's weird, but the `MONGO_HOST` is used to perform liveness check. We can remove it if we set `CHECK_DB_LIVNESS` to `false`.

https://github.com/featbit/featbit/tree/main/modules/data-analytics#mongodb-settings

![image](https://github.com/zeabur/zeabur/assets/34052208/20bcbd1c-9680-4066-9c3f-339f1c389165)

Reply:
https://github.com/zeabur/zeabur/blob/main/prebuilts/mongodb.toml

Basically MONGO_HOST has been implicitly injected into this service. I do not think this is necessary to override this variable again.

Reply:
Aha, so we can safely remove the `MONGO_HOST` and `MONGO_URI` variables?

Reply:
Yeah.

Reply:
resolved in https://github.com/zeabur/zeabur/pull/370